doc/usage/extensions/autosummary.rst

```diff
@@ -136,13 +162,25 @@ also use this new config value:
 .. confval:: autosummary_generate
 
    Boolean indicating whether to scan all found documents for autosummary
-   directives, and to generate stub pages for each.
+   directives, and to generate stub pages for each. It is disabled by default.
 
    Can also be a list of documents for which stub pages should be generated.
 
    The new files will be placed in the directories specified in the
    ``:toctree:`` options of the directives.
 
+Packages can be explored recursively when generating stub pages.
```

Comment:
I think this is a part of description for `autosummary_recursion_limit`. So this should be placed under the entry.

Reply:
removed